gcc/c-family/c-common.h

```diff
@@ -1519,6 +1519,8 @@ namespace selftest {
 
   /* The entrypoint for running all of the above tests.  */
   extern void c_family_tests (void);
+  extern void diagnostic_c_tests (void);
+  extern void opt_problem_cc_tests (void);
```

Comment:
Shouldn't these (remain to) be declared elsewhere?

Reply:
I declared them alongside the other C family tests, and I think it makes more sense now that they're renamed. Please let me know if you were expecting another location :)